### ISSUE TITLE
Missing buses and stops

### ISSUE THREAD

**Issue by vitorpegas95** [carrismetropolitana/schedules-api#32]
Hey all!
First of all, congratulations on the amazing work. This API is superb, very well written and very easy to use.

I'm having a few issues on some buses, routes and stops missing from the API.
Example: 724, 729, 764, "Calvário" stop, any bus stop in Benfica train Station.
Essentialy, any bus without 4 digits, is not present, and perhaps their stations are not present either.

Is there a reason for this, maybe there's a parameter I'm missing or maybe these buses don't have GPS yet?

Thank you :)

**Comment by joao-vasconcelos** [comment]
Olá Vitor! Muito obrigado pelo feedback! Deixo-te aqui a resposta a outro issue com a mesma pergunta: https://github.com/carrismetropolitana/api/issues/6#issuecomment-1739873017

TLDR: Carris Metropolitana e Carris (municipal) não são a mesma empresa.

**Comment by vitorpegas95** [comment]
Realmente esqueci-me de ver os issues closed. Peço desculpa 🙏 
E peço também desculpa por teres já respondido a esta pergunta 3575928x 😅 

Continua o bom trabalho! 
Obrigado

**Comment by joao-vasconcelos** [comment]
Tudo bem! É só aceitar que é confuso 🥲
Obrigado!
